conf/messages

```diff
@@ -35,13 +35,12 @@ landing.start=Start now
 
 ## Successfully enrolled page
 successfullyEnrolled.heading=You have successfully enrolled with S&S GB
-successfullyEnrolled.para.1=Your enrolment will be active in 2 hours.
-successfullyEnrolled.coms=S&S GB handles digital communications between customs administrators and carriers or their appointed representatives.
-successfullyEnrolled.incorporates=S&S GB incorporates the:
-successfullyEnrolled.incorporates.1=lodging, handling and processing of an Entry Summary declaration (sometimes called an ENS) in advance of the arrival of goods into the UK from outside the UK
-successfullyEnrolled.incorporates.2=issuing of a Movement Reference Number (MRN)
-successfullyEnrolled.makeADeclaration=Once your account is active, you can submit an
-successfullyEnrolled.makeADeclaration.linkText=Entry Summary declaration
+successfullyEnrolled.para.1=You can submit an Entry Summary declaration, once your account is active
+successfullyEnrolled.para.1=You will need to use third party software in order to do this
```

Comment:
You are overwriting these messages causing the content to not display correctly 
![Screenshot 2021-01-08 at 13 50 09](https://user-images.githubusercontent.com/6122384/104022746-9a549980-51b8-11eb-878c-d9df16e91c64.png)
Make sure the message Keys are unique for each message for example the first 2 messages could be
`successfullyEnrolled.para.1=You can submit an Entry Summary declaration, once your account is active
successfullyEnrolled.coms=S&S GB handles digital communications between customs administrators and carriers or their appointed representatives.	
successfullyEnrolled.para.2=You will need to use third party software in order to do this`
Then update view to match